.github/workflows/build_test.yaml

```diff
@@ -76,6 +73,7 @@ jobs:
               kokkos: -DKokkos_ENABLE_SERIAL=ON
               kokkos_fft: -DCMAKE_CXX_FLAGS="-Wall -Wextra" -DCMAKE_COMPILE_WARNING_AS_ERROR=ON -DCMAKE_CXX_CLANG_TIDY="clang-tidy;-warnings-as-errors=*"
               benchmark: OFF
+            cmake_build_type: Debug
```

Comment:
Is this flag also used to compile Kokkos ?

Reply:
Yes. Maybe keep using `Release` for `Kokkos`?

Reply:
No I think it is fine then. Debug enables bounds access in Kokkos views